src/Admin/PageAdmin.php

```diff
@@ -402,6 +403,12 @@ protected function configureFormFields(FormMapper $formMapper): void
                             'siteId' => $this->getSubject() ? $this->getSubject()->getSite()->getId() : null,
                         ],
                     ])
+                ->add('targetRedirectType', ChoiceType::class, [
+                        'choices' => [
+                            Response::HTTP_MOVED_PERMANENTLY => $this->trans('choice.redirect_permanent'),
```

Comment:
Please don't use `trans` anymore. It is deprecated: https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Admin/AbstractAdmin.php#L1943

Try to use `translation_domain` / `choice_translation_domain`.

Reply:
Changed it. Used them because they're still used in the [filters](https://github.com/sonata-project/SonataPageBundle/blob/3.x/src/Admin/PageAdmin.php#L306-L307).

Reply:
Sorry, just realised my solution doesn't make sense as the default translation domain in the admin is `messages` and not `SonataPageBundle`. I'm also not sure if I need to use `choice_translation_domain` or `catalogue` as `catalogue` isn't available in the symfony form choice type. I need to take some more time tomorrow to figure this out properly.